scripts/UrlMonitor.php

```diff
@@ -9,6 +9,10 @@
 
 $Monitor = new \UrlMonitor\Monitor();
 
-$Monitor->run($argv[1]);
-
+if (isset($argv[1])) {
+	$domain = filter_var($argv[1], FILTER_SANITIZE_STRING);
+} else {
+	$domain = (string) "";
```

Comment:
My suggestion is an exception 
` throw new RuntimeException("Empty Domain Parameter - Example: php UrlMonitor.php www.github.com' ");`
instead of 
`$domain = (string) "";`

Reply:
Is fixed